packages/ag-charts-community/src/options/series/topology/topologyOptions.ts

```diff
@@ -7,6 +7,8 @@ export type AgTopologySeriesOptions = AgMapSeriesOptions | AgMapMarkerSeriesOpti
 export interface AgBaseTopologyChartOptions {
     /** Series configurations. */
     series?: AgTopologySeriesOptions[];
+    /** Topology to use in all series. */
+    topology?: any;
```

Comment:
Nit: Could we use an alias type called `GeoJson` to give some hint as to what is expected here?

Reply:
Without a peer dep on `geojson`, it's not easily possible. I do want to add some runtime validation